designer/server/src/main/scala/pl/touk/nussknacker/ui/process/test/ScenarioTestService.scala

```diff
@@ -179,10 +178,13 @@ class ScenarioTestService(
 
   private def assignUserFriendlyEditor(uiSourceParameter: UISourceParameters): UISourceParameters = {
     val adaptedParameters = uiSourceParameter.parameters.map { uiParameter =>
-      uiParameter.editor match {
-        case DualParameterEditor(StringParameterEditor, DualEditorMode.RAW)
+      uiParameter.editors match {
```

Comment:
After changing the default for string to spel template, this method is not needed at all. Take a look, the purpose of it was to replace the default combination of editors for string to be more human-friendly. After your changes, a human-friendly editor will be everywhere, as well in scenario testing context.

Reply:
Done